src/components/app-switcher/app-switcher.scss

```diff
@@ -0,0 +1,98 @@
+@import "./../../style-config/colors";
+.carbon-icon__svg--grid {
```

Comment:
This CSS looks as though it belongs to the icon class, so should exist along with any CSS for that component rather than here.